src/main/java/me/gnat008/perworldinventory/groups/Group.java

```diff
@@ -56,9 +60,41 @@ public Group(String name, List<String> worlds, GameMode gameMode) {
      * @param configured If the group is defined in the worlds.yml file.
      */
     public Group(String name, List<String> worlds, GameMode gameMode, boolean configured) {
+        this(name,worlds,null,null,null,null,gameMode,configured);
+    }
+
+
+    /**
+     * Constructor.
+     *
+     * @param name The name of the group.
+     * @param worlds A list of world names in this group.
+     * @param gameMode The default {@link GameMode} for this group.
+     * @param useLastWorld A {@link List} of teleport causes that should map to the last world in group
+     * @param usePosInWorld A {@link List} of teleport causes that should map to the last position in world
+     */
+    public Group(String name, List<String> worlds, List<String> useLastWorld, List<String> useLastPosInGroup, List<String> useLastPosInWorld, String defaultWorld, GameMode gameMode) {
+        this(name,worlds,useLastWorld,useLastPosInGroup,useLastPosInWorld,defaultWorld,gameMode,false);
+    }
+
+    /**
+     * Constructor.
+     *
+     * @param name The name of the group.
+     * @param worlds A list of world names in this group.
+     * @param gameMode The default {@link GameMode} for this group.
+     * @param useLastWorld A {@link List} of teleport causes that should map to the last world in group
+     * @param usePosInWorld A {@link List} of teleport causes that should map to the last position in world
+     * @param configured If the group is defined in the worlds.yml file.
+     */
+    public Group(String name, List<String> worlds, List<String> useLastWorld, List<String> useLastPosInGroup, List<String> useLastPosInWorld, String defaultWorld, GameMode gameMode, boolean configured) {
```

Comment:
Yup, very true. I did it this way to make it more obvious, but I guess it's, in retrospect, very silly.

I thought it'd make more sense if the set matched the external YML files datatype rather than the internal `TeleportCause`, but yeah... That's bonkers.